src/main/java/coffee/order/controller/Customer.java

```diff
@@ -0,0 +1,85 @@
+package coffee.order.controller;
+
+import coffee.order.domain.*;
+import coffee.order.view.InputView;
+import coffee.order.view.OutputView;
+
+import java.util.List;
+
+import static coffee.order.domain.Answer.YES;
+
+public class Customer {
+    public static final int COUNT_INDEX = 1;
+    public static final String NO_VALUE = "";
+    private final Kiosk kiosk = new Kiosk();
+    private final OutputView outputView = new OutputView();
+    private final Cart cart = new Cart();
+
+
+    public void doOrder() {
+        setMenus();
+        addMenus();
+        printCart();
+        useCoupon();
+        showReceipt();
+    }
+
+
+    private void setMenus() {
+        outputView.printMenu(kiosk.showMenus());
+    }
+
+    private void addMenus() {
+        while (true) {
+            List<String> menuInfo = InputView.inputMenu();
+            if (!menuInfo.get(0).equals(NO_VALUE)) {
+                addMenu(menuInfo);
+            } else if (menuInfo.get(0).equals(NO_VALUE)) {
+                break;
+            }
```

Comment:
```suggestion
        boolean equals = menuInfo.get(0).equals(NO_VALUE);
        if (!equals) {
            addMenu(menuInfo);
        } else {
            break;
        }
```
동일한 메서드를 두 차례 이상 호출해야 하는 상황이라면 변수를 활용 하는 것이 좋을 듯 합니다.
그리고 if에서 false 조건으로 분기를 했으므로 다음 분기 조건은 중복된 코드를 작성하지 않아도 될 듯 합니다.